lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -804,12 +808,12 @@ WasmBinaryReader::ReadElementSection()
     for (uint32 i = 0; i < count; ++i)
     {
         uint32 index = LEB128(length); // Table id
-        if (index != 0)
+        if (index != 0 || !(m_module->HasTable() || m_module->HasTableImport()))
```

Comment:
true, but the design says that every module has one default memory/table (not up to one). there is no mention what the state is if there is no explicit table section. so if this is how the spec interpreter behaves then i'm fine deferring to it. just one of those corners that will needed to be worked out in the formal spec